### DISCUSSION TITLE
Consider `[?]` items...

### DISCUSSION THREAD

**Discussion by NiloCK** [jotaen/xit#34] [File Format]
Sometimes I write down something that I **might** need to do in the future, pending outcomes outside of my influence.

This is essentially a pre-flag for `[ ]` items which are more likely than average to become `[~]` items.

**Comment by jotaen** [comment]
Thanks for sharing your idea. I think `[?]` could be a useful status, and it should also be obvious what it means.

The only caveat is that I’d like to keep the overall number of statuses contained to like a handful, to keep things simple. So I first wanted to see whether or what other ideas come up over time, before adding a new one. I think, `[?]` could be a high-up candidate, though.

**Comment by shah** [comment]
Agreed, @NiloCK `[?]` could be a great additional status for individuals but IMHO it's more important in case a TODO list is stored in Git for use by _multiple collaborators_. Some collaborators would find it easier to add a `[?]` as a _suggested_ item and it would be history-tracked as such. 

It could have a very simple workflow - a `[?]` is a potential start state equal to `[ ]` when the submitter (collaborator) is not sure whether it's a real task or a potential task. 

Linters for `xit` could then differentiate their lint output for a _suggested_ or _potential_ task vs. a _decided_ task?

**Comment by clach04** [comment]
Cross ref https://github.com/jotaen/xit/issues/44
